pokemongo_bot/cell_workers/sniper.py

```diff
@@ -210,13 +227,12 @@ class Sniper(BaseTask):
     MIN_SECONDS_ALLOWED_FOR_CELL_CHECK = 10
     MIN_SECONDS_ALLOWED_FOR_REQUESTING_DATA = 5
     MIN_BALLS_FOR_CATCHING = 10
-    MAX_CACHE_LIST_SIZE = 200
+    MAX_CACHE_LIST_SIZE = 500
```

Comment:
You  do realize people bot on raspberry devices, right? Increasing this this high might not be a good idea.